front/pages/w/[wId]/members/index.tsx

```diff
@@ -226,43 +227,14 @@ export default function WorkspaceAdmin({
 
     const [searchText, setSearchText] = useState("");
     const { members, isMembersLoading } = useMembers(owner);
+    console.log(searchText);
     const { invitations, isInvitationsLoading } =
```

Comment:
can we move the SSR in the InvitationsList componenet?

Reply:
Same for Members?

Reply:
Well not really, from what I understood fetching data through `getServerSideProps` is executed at request time of a page, but not components level.

Here seems like retrieving `members` is done via a hook which means it could me moved to `MembersList`, right? But since `members` is also used by the `InviteEmailModal` I would probably leave it here. What do u think?

Reply:
However seems like `invitations` is fetched through a hook and only used by the `InvitationsList` component so I can move the data fetching there 😃